policyengine_us/variables/gov/states/mt/tax/income/credits/ctc/mt_ctc_eligible.py

```diff
@@ -0,0 +1,30 @@
+from policyengine_us.model_api import *
+
+
+class mt_ctc_eligible(Variable):
+    value_type = bool
+    entity = TaxUnit
+    label = "Eligible for the Montana Child Tax Credit"
+    definition_period = YEAR
+    reference = "https://leg.mt.gov/bills/2023/billpdf/HB0268.pdf"
+    defined_for = StateCode.MT
+
+    def formula(tax_unit, period, parameters):
+        p = parameters(
+            period
+        ).gov.states.mt.tax.income.credits.ctc.eligibility.income_limit
+        # CTC limited to filers with AGI at or below $56,000
+        agi = tax_unit("adjusted_gross_income", period)
+        income_eligible = agi <= p.agi
+        # CTC limited to filers with investment income below $10,300
+        investment_income_eligible = (
+            tax_unit("net_investment_income", period) < p.investment
```

Comment:
we need to make a disqualified_income.py variable as stated here:
https://www.law.cornell.edu/uscode/text/26/32#:~:text=In%20the%20case%20of%20an,exceed%20the%20earned%20income%20amount.
make a list parameter with all the components

Reply:
use disqualified income in the variables